### ISSUE TITLE
A11y Fail: Pagination - Need aria-label

### ISSUE THREAD

**Issue by orion-cengage** [cengage/react-magma#1387]
CENG_1959

https://react-magma.cengage.com/version/4.5.0/api/pagination/

### Issue:
The pagination control uses a `<nav>` element but the element does not have a label to differentiate it from other navigation landmarks.

### Suggestion:
Please use `<nav aria-label="pagination">` to label this landmark.

**Comment by silvalaura** [comment]
We should be able to address this by adding `aria-label` to all our examples (it should by default trickle down to the `nav` tag), without explicitly adding a new prop. Or, we can decide that it's important enough that we add this as a prop that defaults to `pagination` but users can customize. Ideally it would be a required prop but that would be a breaking change, so maybe not at this time.

**Comment by chris-cedrone-cengage** [comment]
-Verify the use of aria-current ✅
-Verify that aria-label is visible on the nav property ✅

All set

**Comment by chris-cedrone-cengage** [comment]
`aria-label="pagination"` ✅

All set
